apps/native-component-list/src/navigation/MainTabNavigator.tsx

```diff
@@ -77,8 +83,8 @@ export default function MainTabbedNavigator(props: any) {
       drawerContent={props => <CustomDrawerContent {...props} hideLabels={isTablet} />}
       drawerStyle={{ width: isLargeScreen ? undefined : 64 + left }}
       drawerType="permanent">
-      {Object.keys(Screens).map(name => (
-        <Drawer.Screen
+      {(Object.keys(Screens) as (keyof typeof Screens)[]).map(name => (
```

Comment:
Same here 😉 
I know that you don't like to change both syntax/semantics (js ➡️ ts) as well actual logic in one PR, but at this point I believe it's irrelevant to keep this distinction, because of:
- you can write exactly the same logic using other language constructs
- fixing TS errors sometimes even suggests these better tailored semantics for some cases (like this one 😉)

Reply:
I didn't notice `Object.entries` would solve this problem (my bad, I tried it when `Screens` was still a `Record<…, undefined | …>`, which obviously didn't help at all then), I agree that if a slightly different code produces the same result, is as readable as its previous version and solves a types problem it's worth changing it. 👍